react-app/src/stylesheets/custom.scss

```diff
@@ -0,0 +1,12 @@
+@import './_variables.scss';
+
+$theme-colors: (
+  'primary': $primary,
+  'secondary': $secondary,
+  'info': $info,
+  'success': $success,
+  'warning': $warning,
+  'danger': $danger,
+);
+
+@import '../../node_modules/bootstrap/scss/bootstrap.scss';
```

Comment:
DP2 - We shouldn't be relative pathing to node_modules in any of our JS/SCSS files. SCSS used through CRA supports pathing to the `node_modules` directory using the `~` character. See [here](https://create-react-app.dev/docs/adding-a-sass-stylesheet/)